test/testdata/resolver/sig_misc.rb

```diff
@@ -30,7 +30,7 @@ def f1(x) # error: Type not specified for argument
     T1.new
   end
 
-  sig do params(x: T1).returns(T2) end.hithere # error: Can not call method `hithere` on void type
+  sig do params(x: T1).returns(T2) end.hithere # error: Method `hithere` does not exist on `NilClass`
```

Comment:
It's unfortunate that this error message changes, as `sig` doesn't return a `NilClass` result. This isn't blocking, but it would be great if we could get back to the old error.

Reply:
+1, could you elaborate on this more? Where does `NilClass` come from?

Reply:
This sig gets moved along with the method underneath it and then is replaced with an `EmptyTree`, which evaluates to `NilClass`.